canonical-websites/snapcraft.io
# CLI install command

# Done

Fixes https://github.com/canonical-websites/snapcraft.io/issues/389

- Removed buggy and unsupported 'Install' button
- Added CLI install instruction

# QA

- Pull this branch
- `./run`
- Visit http://0.0.0.0:8004/vlc or visit http://snapcraft.io-pr-419.run.demo.haus/vlc
- Click the copy to clipboard button
- Paste the result somewhere, it should match:
  `sudo snap install vlc`

# Screenshots

## Desktop
![screenshot-2018-3-21 vlc linux software in the snap store 1](https://user-images.githubusercontent.com/479384/37705695-d1da63b2-2cf4-11e8-80dd-cba09c9da8ae.png)

## Mobile
![screenshot-2018-3-21 vlc linux software in the snap store](https://user-images.githubusercontent.com/479384/37705701-d5445120-2cf4-11e8-842e-9a59b263abed.png)
